AQ3DC/AQ3DC.py

```diff
@@ -1688,7 +1757,61 @@ def ComputeMeasure(self, list_measure: list, dic_patient: dict):
                         f"Dont write this measure {measure} for this patient {patient} because is useless measure"
                     )
                     continue
-        return dic_patient__computation
+
+        # for measure in list_measure :
+        #     dic_measurement_sheet = {
+        #     "Patient": [],
+        #     "Landmarks" : [],
+        #     "R-L Component": [],
+        #     "R-L Meaning": [],
+        #     "A-P Component": [],
+        #     "A-P Meaning": [],
+        #     "S-I Component": [],
+        #     "S-I Meaning": [],
+        #     "3D Distance": [],
+        #     "Yaw Component": [],
+        #     "Yaw Meaning": [],
+        #     "Pitch Component": [],
+        #     "Pitch Meaning": [],
+        #     "Roll Component": [],
+        #     "Roll Meaning": [],
+        # }   
+        #     add_in_name_sheet = ''
+        #     if measure["Landmarks"].replace('/','-')[:3] == "Mid" :
+        #         add_in_name_sheet = 'Mid'
+        #     else :
+        #         add_in_name_sheet = measure["Landmarks"].replace('/','-')
+        #     type_measure = dic_short_cut[measure["Type of measurement + time"]] + add_in_name_sheet
+        #     dic_with_all_measurement[type_measure] = dic_measurement_sheet
+        #     for patient, point in dic_patient.items():
+        #         try:
+        #             measure["position"] = point
+        #         except KeyError as key:
+        #             print(f"this landmark {key} doesnt exist for this patient", patient)
+        #             continue
+        #         try:
+        #             measure.computation()
+        #         except ZeroDivisionError as Zero:
+        #             print(
+        #                 f"impossible to compute this measure {measure} for this patient {patient} a reason divide by 0 {Zero}"
+        #             )
+        #             continue
+
+        #         measure.SignManager()
+
+        #         if measure.isUtilMeasure():
+        #             dic_with_all_measurement[type_measure]["Patient"].append(patient)
+        #             for title in list_title:
+        #                 dic_with_all_measurement[type_measure][title].append(measure[title])
+
+        #         else:
+        #             print(
+        #                 f"Dont write this measure {measure} for this patient {patient} because is useless measure"
+        #             )
+        #             continue
```

Comment:
It s a new functionality. I talked with luc and help me do my paper.
I will talk with all the lab, to know if they want to implement this functionality.

This functionality is to create a excel result with one sheet by type of measurement.

Reply:
It may be more appropriate to place the working version of that code in a separate branch rather than a comment... but since you and Luc are both aware of it I think it is fine here.

Thanks for making those other changes!